apps/docs/app/(home)/MyRuntimeProvider.tsx

```
@@ -5,18 +5,49 @@ import {
-      apiKey: process.env["NEXT_PUBLIC_BACKEND_API_KEY"],
-    },
+import {
+  type ChatModelAdapter,
+  type ChatModelRunOptions,
+  useLocalRuntime,
+} from "@assistant-ui/react";
+
+async function* tokenByToken(str: string) {
+  for (const token of str.split(" ")) {
+    await new Promise((resolve) => setTimeout(resolve, Math.random() * 10 + 5));
+    yield token + " ";
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)